src/Tasks/Resources/Strings.resx

```diff
@@ -287,7 +287,7 @@
     <comment>{StrBegin="MSB3030: "} LOCALIZATION: {0} is a number.</comment>
   </data>
   <data name="Copy.FileLocked">
-    <value>File "{0}" is locked by: "{1}"</value>
+    <value>The file is locked by: "{1}"</value>
```

Comment:
Actually if you're still passing two arguments then it has to stay {1} (and {0} argument is never used)